__tests__/server.test.js

```diff
@@ -0,0 +1,56 @@
+const request = require("supertest");
+const server = require("../middleware/server");
+const db = require("../db/connection");
+const seed = require("../db/seeds/seed");
+const testData = require("../db/data/test-data/index.js");
+
+afterAll(() => {
+  return db.end();
+});
+
+beforeEach(() => {
+  return seed(testData);
+});
```

Comment:
Good work implementing these 2 jest lifecycle hooks including the seeding of the `test` data before each test.